Airport Ticket Booking/Airport Ticket Booking/Commands/ManagerCommands/ValidationDetailsCommand.cs

```diff
@@ -0,0 +1,65 @@
+﻿using AirportTicketBooking.DBHandler;
+using AirportTicketBooking.Models;
+using AirportTicketBooking.Validation;
+using System.ComponentModel.DataAnnotations;
+using System.Reflection;
+using System.Text;
+
+namespace AirportTicketBooking.Commands.ManagerCommands
+{
+    internal class ValidationDetailsCommand : ICommandManager
```

Comment:
`DisplayValidationDetailsCommand`

Ok I see, you just return the strings and Print it on the Interface, mmmm I guess it's ok to print to the console here (with providing some abstraction instead of using the console directly).